src/UniversalPackages/Microsoft.Build.UniversalPackages.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>net8.0;net472</TargetFrameworks>
```

Comment:
You sure you don't want to make this netstandard? Huge compat savings in the future to for some light hackery.

Reply:
Multitargeting is pretty clean for MSBuild tasks and doesn't really cause issues. Just adds 1 line to choose the correct task assembly.